<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_input">
     <item>
      <layout class="QFormLayout" name="formLayout_text">
       <item row="0" column="0">
        <widget class="QLabel" name="label_find">
         <property name="text">
          <string>Find what:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_find"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_replace">
         <property name="text">
          <string>Replace with:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_replace"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_match_case">
       <property name="text">
        <string>Match case</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_wrap_around">
       <property name="text">
        <string>Wrap around</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_buttons">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_find_next">
       <property name="text">
        <string>Find Next</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_find_prev">
       <property name="text">
        <string>Find Prev.</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_replace">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_replace_all">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
